contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,57 @@
-= Version Control (Git)
+= Version Control
 :navtitle: Version Control
 
-*Content under construction.*
+Boost uses the Git version control system for its development. This allows multiple contributors to work on the libraries concurrently, while keeping track of the changes that each person makes. Git is a distributed version control system, which means each contributor has their own local copy of the entire project, including its history.
+
+== Library Repositories
+
+The Boost libraries are modular, and each library has its own separate repository.  This makes it easier to work with the entire collection of Boost libraries.
+
+The official Boost repositories are hosted on GitHub at https://github.com/boostorg/[Boost.org]. Each library has its own repository under this organization. For example, https://github.com/boostorg/json for boost:json[], or https://github.com/boostorg/date_time/ for boost:date_time[].
+
+As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, you can make a pull request, which the maintainers of the libraries can then review and potentially merge into the official codebase. Be aware though that Boost has a well-defined workflow for contributors, including guidelines for how to submit changes, how changes are reviewed, and so forth. Refer to xref:getting-involved.adoc[].
```

Comment:
Maybe link to the "Contribute to an Existing Library" subsection of getting started instead?